doc-src/templates/api-versions/model_documentor.rb

```diff
@@ -93,11 +93,19 @@ def initialize(operation_name, operation, api, klass)
     @lines += shapes(api, operation['input']).map {|line| "  " + line }
 
     ## @example tag
-
     @lines << "@example Calling the #{method_name(operation_name)} operation"
```

Comment:
I would suggest moving any hand-written examples above the generated example, since they are likely to be more correct / useful.